user_guide_src/source/outgoing/view_cells/018.php

```diff
@@ -10,6 +10,6 @@ class RecentPostsCell extends Cell
 
     public function mount()
     {
-        $this->posts = model('PostModel')->getRecent();
+        $this->posts = model('PostModel')->orderBy('created_at', 'DESC')->findAll();
```

Comment:
Thank you! but `findAll()` gets all records and these seems too many.
This is `RecentPostsCell` class, so I think we should limit the records.

Reply:
Well, I figured that since this is just for documentation purposes, it shouldn't be an issue. 

I'll push changes based on your suggestion, ASAP.